assignments/prototypes.js

```diff
@@ -10,38 +10,74 @@
   
 /*
   === GameObject ===
-  * createdAt
-  * name
-  * dimensions (These represent the character's size in the video game)
-  * destroy() // prototype method that returns: `${this.name} was removed from the game.`
+  *11 createdAt
+  * 11name
+  * 11dimensions (These represent the character's size in the video game)
+  * 11destroy() // prototype method that returns: `${this.name} was removed from the game.`
+  * 
+  * DONE
 */
+"use strict"
+function GameObject (attributes) {
+  this.createdAt = attributes.createdAt;
+  this.name = attributes.name;
+  this.dimensions = attributes.dimensions;
+}
+
+GameObject.prototype.destroy = function  () {
+  return (`${this.name} was removed from the game`)
+}
```

Comment:
and accessing the parents prototype and adding another method(function attached or built into a object) to it.